js/youtube_audio.js

```diff
@@ -1,35 +1,40 @@
-function makeSetAudioURL(videoElement, url) {
-    function setAudioURL() {
-        if (videoElement.src  != url) {
-            videoElement.pause();
-            videoElement.src = url;
-            videoElement.currentTime = 0;
-            videoElement.play();
-        }
+var oldUrl;
```

Comment:
Not sure, why do you require this.

Reply:
I planned to reset video without reloading. But I give up, don't think it's worth spending much time.
You can delete whatever you don't want.

Reply:
Okay, I will go ahead an merge it after removing this.